samcli/lib/init/template_modifiers/application_insights_template_modifier.py

```diff
@@ -0,0 +1,94 @@
+"""
+Class used to parse and update template when application-insights is enabled
+"""
+import logging
+from typing import Any
+from samcli.lib.init.template_modifiers.cli_template_modifier import TemplateModifier
+
+LOG = logging.getLogger(__name__)
+
+
+class ApplicationInsightsTemplateModifier(TemplateModifier):
```

Comment:
Can we add some description (comment) what is this class supposed to do.

Reply:
Sure, will do

Reply:
@vsukameti were you able to push your changes to address this comment?

Reply:
Yes, I added a one-line description